AAUnlimited/Texts/mod/subtitles.lua

```diff
@@ -1,10 +1,24 @@
 --@INFO Subtitles loader
 
 local _M = {}
---local opts = {
---	{ "maxlines", 4, "Maximum number of lines to show %i[1,10]"},
---	{ "duration", 3, "Subtitles duration %i[1,10]:" },
---}
+local opts = {
+	{ "fontfam", 0, "Font family: %l|Arial|Tahoma|" },
+	{ "fontsize", 24, "Font size, px: %i[1,100]" },
+	{ "lineheight", 120, "Line height, percents: %i[100,300]{Percent of Font size param}" },
+	{ "duration", 5, "Show duration, sec: %i[1,60]" },
```

Comment:
You can skip the max value to avoid having a slider appear

Reply:
Got it.